terraform/files/deploy.sh

```
@@ -0,0 +1,139 @@
+  echo "First-time repository setup..."
+  
+  # Preserve important files (just .env for now as data will be preserved separately)
+  if [ -f "$APP_DIR/.env" ]; then
+    echo "Preserving .env file..."
+    cp "$APP_DIR/.env" /tmp/canadaspends_env_backup
+  fi
+  
+  # For first-time setup, we'll just clean the directory and clone fresh
+  echo "Cleaning directory for clean clone..."
+  shopt -s dotglob  # Include dotfiles in * expansion
+  rm -rf $APP_DIR/* 2>/dev/null || true
```

Shellcheck (0.10.0)

[warning] Use "${var:?}" to ensure this never expands to /* .

(SC2115)